regexes/device/mobiles.yml

```diff
@@ -4381,7 +4381,7 @@ Smartfren:
 
 # Smartisan
 Smartisan:
-  regex: 'SM(?:70[15]|801|901|919)|YQ60[1357][ /;\)]'
+  regex: '(SM|OD)(?:70[15]|801|901|919|105|103)|YQ60[1357][ /;\)]'
```

Comment:
Would be more precise to use `'SM(?:70[15]|801|901|919)|OD10[35]|YQ60[1357][ /;\)]'`
Otherwise strings like `OD70` would match, which might not be correct I guess